pmm/pmm2-server-autobuild.groovy

```diff
@@ -106,13 +106,18 @@ pipeline {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh '''
                         sg docker -c "
+                            set -o errexit
+                            set -o xtrace
+
                             export PATH=$PATH:$(pwd -P)/build/bin
 
                             # 1st-party
                             build-server-rpm percona-dashboards grafana-dashboards
-                            build-server-rpm pmm-manage
+                            ### missing rpm spec in PMM-2.0
+                            # build-server-rpm pmm-manage
```

Comment:
why?

Reply:
We don't have those components in PMM 2.x. Those lines should be removed, not commented-out

Reply:
> Those lines should be removed, not commented-out